### PR TITLE
Glowbug

### PR DESCRIPTION
**Pull request by shb46**
Initial pull request for the Glowbug mission. Contains Glowbug-specific schema, Glowbug GCN schema and example.

### CODE REVIEW

File: gcn/notices/glowbug/specific.schema.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      "items": {
+        "type": "string",
+        "enum": ["off", "on", "triggered"]
+      },
+      "min_items": 15,
+      "max_items": 15,
+      "description": "Detector statuses at trigger time. Detectors: 0..11 - CsI; 12, 13 - CLLB; 14 - SSA."
+    }
+  }
+}
```
Comment (jracusin) [inline gcn/notices/glowbug/specific.schema.json:19]: We decided after our meeting last week that we do not want to use `specific.schema.json`, but rather add this at the bottom of `alert.schema.json`.

Comment (shb46) [reply gcn/notices/glowbug/specific.schema.json]: Judy, I modified it as you requested. Please see the latest commit.

File: gcn/notices/glowbug/alert.example.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://gcn.nasa.gov/schema/gcn/notices/glowbug/alert.schema.json",
```
Comment (dakota002) [inline gcn/notices/glowbug/alert.example.json:3]:
```suggestion
```

File: gcn/notices/glowbug/alert.example.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
```
Comment (dakota002) [inline gcn/notices/glowbug/alert.example.json:2]:
```suggestion
  "$schema": "https://gcn.nasa.gov/schema/gcn/notices/glowbug/alert.schema.json",
```

File: gcn/notices/glowbug/alert.schema.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    {
+      "type": "object",
+      "properties": {
+        "detector_statuses": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": ["off", "on", "triggered"]
+          },
+          "min_items": 15,
```
Comment (dakota002) [inline gcn/notices/glowbug/alert.schema.json:39]:
```suggestion
          "minItems": 15,
```

File: gcn/notices/glowbug/alert.schema.json [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+          "max_items": 15,
```
Comment (dakota002) [inline gcn/notices/glowbug/alert.schema.json:40]:
```suggestion
          "maxItems": 15,
```

**Pull request merged**